platform/security/security-oidc-realm/src/main/java/org/codice/ddf/security/oidc/realm/OidcCredentialsResolver.java

```diff
@@ -123,6 +139,7 @@ public void validate(OidcCredentials credentials, WebContext webContext) {
 
           JWT idToken = userInfoSuccessResponse.getUserInfoJWT();
           if (idToken == null && userInfoSuccessResponse.getUserInfo().toJWTClaimsSet() != null) {
+            // TODO validate user info if signed
```

Comment:
We already do this at line 146 no need for this comment

Reply:
Oh ya I wasn't sure about this one. When we get signed user info back from the IdP, is it guaranteed to be in a JWT? Because if we get signed user info in the form of JSON, right now we are just throwing that into a JWT payload (losing the signature information) for later validation, which wouldn't end up validating a signed JSON user info response.

But if the IdP never sends over a signed JSON response (IOW if the signed responses are always JWT), then we don't have to have this validation stuff here 👍

Reply:
If it's signed it is guaranteed to be a JWT 
Spec : `If the UserInfo Response is signed and/or encrypted, then the Claims are returned in a JWT and the content-type MUST be application/jwt.`